cfgov/v1/atomic_elements/organisms.py

```diff
@@ -170,6 +170,46 @@ class Meta:
         label = ' '
 
 
+class BureaStructurePosition(blocks.StructBlock):
+    name = blocks.CharBlock()
+    title = blocks.StructBlock([
+        ('line_1', blocks.CharBlock(required=False)),
```

Comment:
What are Lines 1 and 2 meant to represent? (Maybe this doesn't matter if it's clear to content editors, just curious)